java/src/plume/WeakHasherMap.java

```diff
@@ -117,7 +118,9 @@
  * @see		java.lang.ref.WeakReference
  */
 
-@SuppressWarnings({"nullness", "keyfor", "interning", "purity"}) // old, non-typesafe Sun code, not worth annotating or checking
+@SuppressWarnings({"nullness", "keyfor", "interning", "purity"}) // old, non-typesafe Sun code, not worth annotating or checking.
+// Annotated for lock checking as an experiment in annotating non-typesafe code.
```

Comment:
I don't think this was the best use of limited time.  We already tell users to use proper generics types and don't want to encourage them otherwise.  I would be inclined to discard the changes to this class, though retaining the annotations is also OK now that the work is done.